src/pages/settings/Profile/TimezoneSelectPage.js

```diff
@@ -36,18 +37,21 @@ const getUserTimezone = (currentUserPersonalDetails) => lodashGet(currentUserPer
 function TimezoneSelectPage(props) {
     const {translate} = useLocalize();
     const timezone = getUserTimezone(props.currentUserPersonalDetails);
-    const allTimezones = useRef(
-        _.chain(TIMEZONES)
-            .filter((tz) => !tz.startsWith('Etc/GMT'))
-            .map((text) => ({
-                text,
-                keyForList: getKey(text),
-                isSelected: text === timezone.selected,
-            }))
-            .value(),
+    const allTimezones = useInitialValue(
+        () =>
+            _.chain(TIMEZONES)
+                .filter((tz) => !tz.startsWith('Etc/GMT'))
+                .map((text) => ({
+                    text,
+                    keyForList: getKey(text),
+                    isSelected: text === timezone.selected,
+                }))
+                .value(),
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+        [],
```

Comment:
Remove these two lines

Reply:
good catch, it was leftover from previous version 👍